roles/common/files/uug_ansible_wrapper.py

```diff
@@ -646,6 +652,42 @@ def unable_to_detect_branch():
         USER_CONFIG['git_branch'] = "master"
         logging.info("Release set to master")
 
+def warn_wrong_release_branch(parent):
+    """
+    Displays a dialog to warn the user that their current branch in the configuration tool does not
+    match their os rlease version.
+    """
+
+    logging.info("The chosen branch is < OS releae. Warning user about switching branches")
+    warning_prompt = (
+        "You are using a version of the configuration tool meant for a different release of Linux Mint. " 
+        "You should consider upgrading braches. The correct configuration settings for your release version of Linux Mint is:"
+        "\nRelease: %(0)s and URL: %(1)s" % {
+            '0': get_distro_release_name(),
+            '1': USER_CONFIG['git_url']
+        }
+    )
+    show_dialog(parent, Gtk.MessageType.WARNING, Gtk.ButtonsType.OK,
+                           "Current branch is outdated", warning_prompt)
+
+def warn_using_master_branch(parent):
+    """
+    Displays a dialog to warn the user that their current branch in the configuration tool is the development branch master.
+    """
+
+    logging.info("The chosen branch is the development bracnch master. Warning user about switching branches")
+    warning_prompt = (
+        "You are currently an unstable development branch (master) of the configuration tool. "
+        "You should consider switching to the release branch for your Linux Mint version. "
+        "The correct configuration settings for your release version of Linux Mint is:" 
+        "\nRelease: %(0)s and URL: %(1)s" % {
+            '0': get_distro_release_name(),
+            '1': USER_CONFIG['git_url']
+        }
+    )
+    show_dialog(parent, Gtk.MessageType.WARNING, Gtk.ButtonsType.OK,
+                           "Current branch is development branch master", warning_prompt)
```

Comment:
I'd like to see us become a little more consistent around syntax more like the following when we wrap lines:

```
func_name(
    param1,
    param2,
    param3,
)
```

OR:

```
func_name(
    param1, param2, param3,
    param4, param5, param6,
)
```

Whichever feels more appropriate in the moment. I think maybe the second would be best here? I'll leave it up to you to decide.

Anyway, note the line break after `(` and before `)` and how the first character of each parameter lines up.